stories/translator/doc-elements.tsx

```diff
@@ -32,28 +37,89 @@ const FormPreview = ({ data }: IProps) => {
     );
 };
 
-const SchemaView = ({ data, onChange }: ISchemaViewProps) => {
-    const [schema, setSchema] = useState("");
+const SchemaView = ({ data, onChange, formBuilderRef }: ISchemaViewProps) => {
+    // ===========================================================================
+    // CONST, STATE, REFS
+    // ===========================================================================
+    const [hasError, setHasError] = useState<boolean>(false);
+    const {
```

Comment:
do we really need to use react-hook-form here? I have the impression that we only need submit / reset the values, it will be good to keep the example short and easy to understand.